addons/okr-daily-script.kp/knowledge.md

```diff
@@ -176,33 +176,46 @@ ms_install_days = ms\
 # combine data
 current = count_by_client_day\
   .join(ms_install_days, on='client_id', how='left')\
-  .join(ms_has_theme, on='client_id', how='left')\
+  .join(ms_has_theme, on=['client_id', 'normalized_channel'], how='left')\
   .drop('submission_date_s3')
 
 
 # add total number of distinct clients per day
 # and total number of ditscint clients with a custom theme
-n_clients = ms.select('client_id').distinct().count()
+n_clients = ms.select(['client_id', 'normalized_channel']).distinct()\
+               .groupby('normalized_channel').count()\
```

Comment:
I see your point, but I think its best to keep just the distinct (`client_id`, `normalized_channel`) pairs since downstream we facet by channel and won't be doing any aggregations on the whole. (really we just want to isolate release users, but it's nice to have the other channels). Clients that use multiple channels may do so for add-on related reasons (or may use use add-ons differently depending on channel), so I think it's important to include all clients.